src/scripts/cfo.zig

```diff
@@ -186,8 +187,12 @@ fn run_fuzzers(
                     log.debug("will start '{s}'", .{command});
                     const child = try shell.spawn_options(
```

Comment:
but, sadly, we can't use it here, because we need a child...

It's on my todo list to re-visint shell.zig and try to come up with somewhat more orthogonal set of APIs. I like _using_ shell.zig, but the implementaiton is almost 1000 lines of mostly copy-paste :(